From: Dan Williams <dan.j.williams@intel.com>
Date: Thu, 7 Apr 2016 19:58:44 -0700
Subject: libnvdimm: fix smart data retrieval
Git-commit: 211291126698c8f047617565b2e2e7f822f86354
Patch-mainline: v4.6-rc3
References: FATE#319858

It appears that smart data retrieval has been broken the since the
initial implementation.  Fix the payload size to be 128-bytes per the
specification.

Cc: <stable@vger.kernel.org>
Signed-off-by: Dan Williams <dan.j.williams@intel.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.com>
---
 drivers/nvdimm/bus.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/nvdimm/bus.c
+++ b/drivers/nvdimm/bus.c
@@ -411,7 +411,7 @@ static const struct nd_cmd_desc __nd_cmd
 	},
 	[ND_CMD_SMART_THRESHOLD] = {
 		.out_num = 2,
-		.out_sizes = { 4, 8, },
+		.out_sizes = { 4, 128, },
 	},
 	[ND_CMD_DIMM_FLAGS] = {
 		.out_num = 2,
